steps/container_steps.py

```diff
@@ -253,7 +260,7 @@ def run_command_and_detach(context, cmd):
 @then(u'run {cmd} in container and check its output for {output_phrase}')
 @then(u'run {cmd} in container and check its output contains {output_phrase}')
 @then(u'run {cmd} in container')
-def run_command_expect_message(context, cmd, output_phrase, timeout=80):
+def run_command_expect_message(context, cmd, output_phrase=None, timeout=80):
```

Comment:
I don't think this should be touched, instead: `run {cmd} in container once` should be used.

Reply:
This method is specifically about validating output of some command.

Reply:
ok, I have tried with '...container once' and it seems to work fine without the commit which introduces it. That commit also had a None check I added, but it does not look like it is needed for my present use

Reply:
Commit removed